shared/@hedvig-ui/JsonSchemaForm/json-schema-form.tsx

```diff
@@ -86,11 +85,15 @@ const ArrayFieldTemplate: React.FC<ArrayFieldTemplateProps> = ({
           <ItemWrapper key={element.key}>
             <ItemTitleWrapper>
               <strong>{index + 1}.</strong>
-              <ItemRemoveButton onClick={element.onDropIndexClick(index)}>
+              <Button
+                variation={'icon'}
+                style={{ paddingRight: '0.5em' }}
```

Comment:
Do we want `em` over `rem`?  I don't remember when to use what 🤔 
```suggestion
                style={{ paddingRight: '0.5rem' }}
```

Reply:
Haha honestly no idea, seems to vary [https://css-tricks.com/confused-rem-em/](https://css-tricks.com/confused-rem-em/)

Reply:
I almost exclusively go for `rem` 

> * Pixels are ignorant, don’t use them.
> * Use REMs for sizes and spacing.
> * Use EMs for media queries.

https://engageinteractive.co.uk/blog/em-vs-rem-vs-px